### PR TITLE
Refactor test scripts in npm

### PR DESCRIPTION
**Pull request by koladilip**
Reuse test commands declared in combined tests.

## Description of the change
This change only minor refactor to test scripts and no behaviour change.
> Description here

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1](Reuse individual test scripts in combined test) 

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "node-fetch": "^2.6.7",
     "object-hash": "^2.2.0",
     "parse-static-imports": "^1.1.0",
-    "pprof": "^3.2.0",
```
Comment (saikumarrs) [inline package.json:66]: Is this change required?

Comment (koladilip) [reply package.json]: No reverted, done for testing and no longer needed.

**Review by saikumarrs — APPROVED** [review]

**Review by sanpj2292 — APPROVED** [review]
